### DISCUSSION TITLE
Nhost.io evaluation

### DISCUSSION THREAD

**Discussion by shyndman** [DevsHelpDevs/DevsHelpDevs#10] [Q&A] [answered]
Decided to split this out from https://github.com/DevsHelpDevs/DevsHelpDevs/discussions/5 so we can center the discussion around this particular choice.

Thomas suggested we look at nhost.io to see whether it would meet our backend needs. I suspect it will, but wanted to point out a few details that are worth consideration:

### Deployment
https://docs.nhost.io/quick-start/deploy-app

The suggestion here is that another service be used to host the frontend. Fine, but noteworthy.

nhost does not appear to support custom domains (although...there is a hint that it may [on this page](https://docs.nhost.io/custom-api)), and if that's the case, returning the base HTML from serverless endpoints will not be appropriate.

❓**Question:** If we go with nhost, where do we host the frontend?

### Serverless
https://docs.nhost.io/custom-api

Serverless support is in beta. The most notable omission judging by the docs is the lack of logging, which may not be a problem initially (especially if we don't have much need for custom APIs to begin with), but will become an issue if the project increases in complexity.

### Geography
I can't find any information about where nhost hosts their backends. As discussed on Zoom, not likely to be a huge issue, but personally I'd like to know. (I'll ping them about it)

### Relational Data
The backend is PostgreSQL. Now I personally like relational data, and Postgres, but I'm curious about the level of familiarity people have with RDBMSes.

### Next steps

I'd love to hear peoples' thoughts on this points, but also, ❓**did I miss anything?**

I'm going to build something simple with the tool and see how it works in more detail. I'll post back when I know more.

**Comment by elitan** [comment, marked as answer]
Hey. Johan here (co-founder of Nhost). Happy to share some light on some of the questions:

> nhost does not appear to support custom domains

We actually do. It's just not public yet. Let me know if this is an issue and I'll have it done for your project.

> Serverless support is in beta. The most notable omission judging by the docs is the lack of logging.

Will come in the future. We have logs on everything, we just need to store it more structural and pipe it to our Nhost console. Timeframe: ~2-4 months from now.

> I can't find any information about where nhost hosts their backends.

Amsterdam, London, Bangalore, New York, San Francisco. 

---

Side note: The reason fairly small things will take such a long time right now is because we're upgrading our infrastructure which will take a few months to get in place.

**Reply by shyndman** [reply]
Thanks Johan!

**Reply by StephenLadoye** [reply]
Kudos to Johan, you said few months. For how long specifically?

**Reply by elitan** [reply]
We hope to have the first version ready end of April.

**Reply by StephenLadoye** [reply]
Okay
